Py/DICOMSegmentationPlugin.py

```diff
@@ -143,54 +143,67 @@ def load(self,loadable):
       (success,labelNode) = slicer.util.loadLabelVolume(labelFileName, returnNode=True)
 
       # Initialize color and terminology from .info file
-      # Format of the .info file:
+      # See SEG2NRRD.cxx for how it's written.
+      # Format of the .info file (no leading spaces, modifiers are optionally written if present):

```

Comment:
Makes sense, I'll initialize all the strings. 
In future if you've got feedback on a pull request that requires changes, it would be best if you held off on merging as it's easier for me to add another commit to the topic branch and squash them once I've addressed all your concerns.